tests/pages/pmmSettingsPage.js

```diff
@@ -433,6 +433,17 @@ module.exports = {
     );
   },
 
+  async verifySettingsValue(field, expectedValue) {
```

Comment:
isn't it a generic method? I mean I can pass any locator as a field and expected value, right?

Reply:
yes you are right, nothing with settings it can be used with any field and page